packages/component-library/package.json

```diff
@@ -32,6 +32,7 @@
   },
   "dependencies": {
     "@kaizen/component-base": "^1.1.1",
+    "@kaizen/component-library": "16.1.4",
```

Comment:
🤔 Why does it have a dependency on itself?

Reply:
silly me was importing through the package rather than the file paths. I have updated now